### ISSUE TITLE
MyPy linter assumes all output from CLI is in the same format

### ISSUE THREAD

**Issue by ollipasanen** [DamnWidget/anaconda#847]
## Expected Behaviour

MyPy linter should show errors after linting a file.

## Actual Behaviour

No mypy linting warnings are shown, MyPy linter throws an error with no traceback in anaconda_jsonserver.log
```
2019-12-23 12:20:29,888: ERROR   : list index out of range
```

## Steps to Reproduce

- Install latest stable mypy via python3-pip
- Install Anaconda
- Enable "mypy": true in anaconda settings
- Try to lint a python file with typing errors, causing mypy to produce at least one warning

## ST3, Anaconda and OS versions

- Sublime Text Build 3211
- Anaconda v2.1.34
- Python 3.7.3
- Ubuntu 18.04
- mypy 0.761

## ST3 Console Logs

Nothing abnormal.

## Anaconda's JsonServer Logs
**Note**: Anaconda's JsonServer logs can be found in:
* Linux: ~/.local/share/anaconda/logs
* OS X: ~/Library/Logs/anaconda
* Windows: %APPDATA\\Anaconda\\Logs

After enabling exc_info in the Exception handler:

```
Traceback (most recent call last):
  File "/home/X/.config/sublime-text-3/Packages/Anaconda/anaconda_lib/linting/anaconda_mypy.py", line 65, in execute
    errors = self.check_source()
  File "/home/X/.config/sublime-text-3/Packages/Anaconda/anaconda_lib/linting/anaconda_mypy.py", line 122, in check_source
    'lineno': int(data[1]),
IndexError: list index out of range
```

This is caused by MyPy CLI producing these types of lines, which are not handled in anaconda_mypy.py correctly

```
Found 3 errors in 1 file (checked 1 source file)
Success: no issues found in 1 source file
```

**Comment by bharel** [comment]
Is the break because of mypy or anaconda? I tried downgrading both and it didn't seem to work.

**Comment by ollipasanen** [comment]
mypy works as it should, but the anaconda mypy plugin is not parsing the CLI output correctly. It assumes all output lines are formatted the same even though the last line is the summary.

**Comment by J-A-M-E-5** [comment]
Ugly fix in anaconda_lib/linting/anaconda_mypy.py. It works.

```
        errors = []
        for line in out.splitlines():
            if line.startswith('Found '):  #  Add this line
                break                      # And this one
```

I suspect there might be a mypy option in latest version to not display this summary line - maybe that is the correct fix?

**Comment by ZackSheffieldSEL** [comment]
I'm having a similar issue. 

mypy on the command line shows this:

```
fixedpoint_lib\q_class.py:15: error: Skipping analyzing 'fixedpoint': found module but no type hints or library stubs
fixedpoint_lib\q_class.py:15: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
```

but in the ST3 status bar it shows this:

![image](https://user-images.githubusercontent.com/4326326/80749220-fa982500-8ada-11ea-8b9d-61e73b5cac0e.png)

Seems like the colon in the URL is delimiting the message in the status bar.

**Comment by scuml** [comment]
The latest master should fix this.  It hasn't been tagged for release quite yet.  If you'd like, you can download the master branch, copy over `anaconda_lib` to your `Packages/Anaconda/` directory, and see if that resolves the problem in the meantime.

**Comment by ZackSheffieldSEL** [comment]
That did the trick!
![image](https://user-images.githubusercontent.com/4326326/80755193-3a640a00-8ae5-11ea-9a9a-0d254c2db93c.png)
Thanks

**Comment by DamnWidget** [comment]
Will make a release tomorrow or Saturday

**Comment by DamnWidget** [comment]
This should be fixed as per @scuml changes introduced on `v2.1.36`, please, feel free to reopen if necessary
